File: 04 Technical/02 Server/01 Logics/01 Python/01 General/03 Duplicate Items 2.py
# Remove Duplicate Items in a List

given_list = [5, 1, 2, 2, 3, 3, 3, 4, 4, 4, 4]  # list(map(int, input().split()))
print(given_list)

print("----------------")

unique_items_list = []
for each_item in given_list:
    if each_item in unique_items_list:
        pass
    else:
        unique_items_list.append(each_item)


print(unique_items_list)  # [5, 1, 2, 3, 4]

print("====================")

File: 04 Technical/01 Client/01 Logics/01 Javascript/03 Duplicate Items 2.js
// Remove Duplicate Items in a List

let given_list = [6, 1, 2, 3, 3, 4, 5, 5, 5];
console.log(given_list);

console.log('----------------');

unique_items_list = [];
for (let item of given_list) {
	if (!unique_items_list.includes(item)) {
		unique_items_list.push(item);
	}
}

console.log(unique_items_list);

console.log('====================');
